regulations/static/regulations/css/scss/_typography.scss

```diff
@@ -300,17 +300,17 @@ a:visited {
 }
 
 .btn, .btn:link, .btn:visited {
-  background-color: #0072ce;
+  background-color: $action_color;
   color: $white;
 }
 
 .btn.hover, .btn:hover {
-  background-color: #328ed8;
+  background-color: $action_color_dark;
 }
 
 .btn.focus, .btn:focus {
-  background-color: #328ed8;
-  outline: #0072ce dotted 1px;
+  background-color: $action_color_dark;
+  outline: $action_color dotted 1px;
```

Comment:
Hm. These colors are slightly different, but I'm not seeing `btn` as a class name in the templates anywhere. Would you mind verifying it's not current in use, and if not, just deleting these rules altogether?